src/pages/workspace/upgrade/WorkspaceUpgradePage.tsx

```diff
@@ -47,6 +47,9 @@ function WorkspaceUpgradePage({route}: WorkspaceUpgradePageProps) {
             case CONST.UPGRADE_FEATURE_INTRO_MAPPING.reportFields.id:
                 Policy.enablePolicyReportFields(policyID, true, true);
                 return Navigation.navigate(ROUTES.WORKSPACE_MORE_FEATURES.getRoute(policyID));
+            case CONST.UPGRADE_FEATURE_INTRO_MAPPING.rules.id:
+                Policy.enablePolicyRules(policyID, true, true);
+                return Navigation.navigate(ROUTES.WORKSPACE_MORE_FEATURES.getRoute(policyID));
```

Comment:
Hmm this is violating the 1:1:1 principle now though... if we want this it should happen inside `upgradeToCorporate` and would need to be a backend change.

Reply:
Oh maybe not because there is a "Got it, thanks" screen 😄  nvm